index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+</head>
+<body>
+    <!-- Aquí va el contenido de tu CV -->
+     <header>
+     <h1>Gabriela Bejarano Pérez</h1>
+     <h2>Información Personal</h2>
+     <ul>
+        <li>gabrielabejarano09@gmail.com</li>
+        <li>314 2425931</li>
+        <li>Diagonal 4b bis 29 23</li>
+     </ul>
+    </header>
+    <main>
+    <h2>Educación</h2>
+     <h3>Bachiller</h3>
+     <p> Acellus Academy | 2020-2023, Kansas City, Missouri</p>
+     <h3>Programacion y Tecnologias Digitales</h3>
+     <p> Universidad Tecnologia de Pereira | Ministerio de las Tecnologias | 2022, Bogotá, Colombia </p>
+     <h3>Ingenieria Informática</h3>
+     <p> Universidad de la Sabana | 2023-Cursando Actualmente | Tercer Semestre, Chía, Colombia</p>
+     <h2>Experiencia Laboral</h2>
+     <h3>Proyecto Biblioteca en Java</h3>
```

Comment:
```suggestion
    <section>
       <h3>Bachiller</h3>
       <ul>
           <li><p> Acellus Academy | 2020-2023, Kansas City, Missouri</p></li>
       </ul>
       
       <h3>Programacion y Tecnologias Digitales</h3>
       <ul>
           <li> <p> Universidad Tecnologia de Pereira | Ministerio de las Tecnologias | 2022, Bogotá, Colombia </p></li>
       </ul>
       
       <h3>Ingenieria Informática</h3>
       <ul>
           <li>       <p> Universidad de la Sabana | 2023-Cursando Actualmente | Tercer Semestre, Chía, Colombia</p>
</li>
       </ul>
       
       
  
       <h2>Experiencia Laboral</h2>
          <ul>
           <li><p> Proyecto Biblioteca en Java</p></li>
       </ul>
     </section>
```

De esta manaera te aseguras que todo tu código se muestre como una lista de listas, lo que es más fácil de leer